src/nplinker/genomics/mibig/mibig_metadata.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+
+class MibigMetadata():
+
+    def __init__(self, file):
+        """To represent the MIBiG BGC metadata (json file)
+
+        Args:
+            file(str): Path to the json file of MIBiG BGC metadata
+
+        Examples:
+            >>> metadata = MibigMetadata("/data/BGC0000001.json")
+        """
+        self.file = file
+        with open(self.file, "rb") as f:
+            self.metadata = json.load(f)
+
+    @property
+    def mibig_accession(self):
+        """Get value of metadata item 'mibig_accession'"""
+        return MibigMetadata.get_mibig_accession(self.metadata)
+
+    @property
+    def biosyn_class(self):
+        """Get value of metadata item 'biosyn_class'"""
+        return MibigMetadata.get_biosyn_class(self.metadata)
```

Comment:
Is the biosyn class a known term in the field or something more special? If so, would be nice to have a short explanation or link here.

Reply:
yes, it's a known term in Mibig metadata, it means "biosynthetic class(es)". I also found it not clear, will add a short explanation for it.